README.md

```diff
@@ -6,8 +6,7 @@ Module contract code is located [here](./src/COWFeeModule.sol). The test for it
 is located [here](./test/COWFeeModule.t.sol).
 
 The driver script is located at [`index.ts`](./index.ts). All other code is in the [`ts`](./ts)
-directory. It uses [`Ethplorer`](https://ethplorer.io) and [`Blockscout`](https://gnosis.blockscout.com) APIs
-to determine the different tokens held by the Settlement contract. It then filters them on the basis of:
+directory. It can use either [`Ethplorer`](https://ethplorer.io), [`Blockscout`](https://gnosis.blockscout.com) APIs or it can directly get the information on-chain using Settlement `Trade` events to determine the different tokens held by the Settlement contract. It then filters them on the basis of:
```

Comment:
Reading what the script does, I noticed that this description was incomplete. The script knows also how to detect tokens reading from the blockchain events.